src/modules/dashboard/components/ColonyHome/ColonyHome.css

```diff
@@ -22,9 +37,17 @@
 }
```

Comment:
do we want to keep empty class name?

Reply:
Haha! That's not empty, just that Github tricked you :)

See this:
![Screenshot from 2020-11-12 20-00-33](https://user-images.githubusercontent.com/1193222/98977846-c093eb00-2521-11eb-8182-17c5ab80c216.png)

It means it hid lines of code it deemed not important. 

If you actually look at the file, you'll see that the left aside has some `padding`:
https://github.com/JoinColony/colonyDapp/blob/b4502176d21c36a3f7c5dae08e00ec269ce7b213/src/modules/dashboard/components/ColonyHome/ColonyHome.css#L31-L33